CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # 1.9.0
 * `savename` now has the `ignore` option.
+* New function `struct2ntuple` that converts a struct to a NamedTuple (for saving)
```

Comment:
since this is a new function added and exported, we have to increment the _minor_ version to 1.10. Please do this in Project.toml and move this part into a new section for version 1.10.

Reply:
Edited Project.toml and updated changelog in dac7bc8